File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Exercises/src/P09_CountTheIntegers.java
import java.util.Scanner;
public class P09_CountTheIntegers {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        int i = 0;
        try {
            for(i = 0; i <= 100; i++) {
                int count = Integer.parseInt(console.nextLine());
            }
        } catch(NumberFormatException hex) {
            System.out.print(i);
        }
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Exercises/P09_CountTheIntegers/P09_CountTheIntegers.cs
using System;

namespace P09_CountTheIntegers
{
    class P09_CountTheIntegers
    {
        static void Main(string[] args)
        {
            int i = 0;
            try
            {
                for (i = 0; i <= 100; i++)
                {
                    int count = int.Parse(Console.ReadLine());
                }
            }
            catch (Exception hex)
            {
                Console.WriteLine(i);
            }
        }
    }
}
